### PR TITLE
[3/11] Implement new header in Packages pages

### PR DESCRIPTION
**Pull request by rcorral**
---
⚠️   _This branch depends on features that will be introduced with #1482_

---

### CODE REVIEW

**Comment by philipnrmn** [comment]
Rebased with the latest state of #1482

**Comment by philipnrmn** [comment]
Looks great! I'm happy with the code, one question about the breadcrumbs - do you think this should be `Packages > Installed` rather than just `Installed Packages`?

screen shot 2016-12-04 at 9 07 56 pm

**Comment by rcorral** [comment]
Good question. Let's ask design, but not let it block the PR.

**Comment by ashenden** [comment]
@philipnrmn is correct.  Sections like "Packages" and "Installed" are really tier-1 sections, but we are choosing to nest them under the "Packages" nav item from a category POV.  So, these sections would read "Installed", not "Packages > Installed" -- or "Users", not "Organization > Users".  They would all use the icon of their category.

BTW, the naming should be 1:1 with the sidebar label.  So "Installed" not "Installed Packages".  We will likely tweak these names over the weeks leading up to GA, so be prepared for that.

File: src/js/pages/UniversePage.js [hunk 1]
```diff
@@ -52,13 +51,7 @@ class UniversePage extends mixin(TabsMixin) {
   }
 
   render() {
-    return (
-      <Page
```
Comment (nLight) [inline src/js/pages/UniversePage.js]: Having deleted this we can also delete all the methods from the class and inherit it directly from `React.Component` I tried and it still worked :)

File: src/js/pages/universe/PackagesTab.js [hunk 2]
```diff
@@ -13,9 +13,18 @@ import FilterInputText from '../../components/FilterInputText';
 import Image from '../../components/Image';
 import InstallPackageModal from '../../components/modals/InstallPackageModal';
 import Loader from '../../components/Loader';
+import Page from '../../components/Page';
 import Panel from '../../components/Panel';
 import StringUtil from '../../utils/StringUtil';
 
+const UniverseBreadcrumbs = () => {
+  const crumbs = [
+    <Link to="universe/packages" key={-1}>Universe</Link>
```
Comment (nLight) [inline src/js/pages/universe/PackagesTab.js]: Going back to John's comment, do we want to change it to `Packages` so it matches with the Sidebar link?

Comment (jfurrow) [inline src/js/pages/universe/PackagesTab.js]: I made this change :)

File: src/js/pages/UniversePage.js (hunk 1 above)
Comment (jfurrow) [inline src/js/pages/UniversePage.js]: Done! Good catch :)

**Comment by jfurrow** [comment]
I implemented both of @nLight's requested changes, so dismissing his review!

This looks & works great!

**Comment by MatApple** [comment]
retest this please

**Comment by MatApple** [comment]
The Jenkins reporter seems to be hanging on this PR. Could be due to the Jenkins maintenance that just wrapped up. Tests are passing so I'm going to merge:

**Pull request merged**